src/styles/blocks/card.scss

```diff
@@ -0,0 +1,119 @@
+.card {
+  display: flex;
+  justify-content: center;
+  box-sizing: border-box;
+  flex-wrap: wrap;
+
+  padding: 16px;
+
+  width: 200px;
+  height: 406px;
+  border-radius: 5px;
+  border: 1px solid $card-border-color;
+
+  background-color: $card-bg-color;
+
+  &__content {
+    display: flex;
+
+    padding: 16px 3px;
+
+    width: 164px;
+    height: 134px;
+  }
+
+  &__nameofproduct {
+    display: flex;
+
+    padding-top: 24px;
+    padding-bottom: 2px;
+
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+
+    color: $main-text-color;
+  }
+
+  &__productcode {
+    display: block;
+
+    width: 100%;
+    padding-top: 2px;
+    padding-bottom: 8px;
```

Comment:
you have a lot of micro-indents.
use 1-sided margins for all indents.
For example:

```
< -- block -- > // title
-- margin-bottom --
< -- block -- > // code
-- margin-bottom --
```
use this logic in all the cases